src/styles/blocks/buy-reasons.scss

```diff
@@ -0,0 +1,63 @@
+@import "../utils/vars";
+@import "../utils/mixins";
+
+.buy-reasons {
+  @include set-horizontal-padding;
+
+  &__ {
```

Comment:
Please change '&__' to '&__element', where 'element' represents the name of the desired element inside the block. This will provide better readability and follow the BEM naming convention.

This convention looks weird and unclear a little :)

Reply:
I changed convention to  '&__element'.